includes/Gateways/Cash.php

```diff
@@ -103,7 +103,6 @@ public function process_payment( $order_id ) {
         $order->update_status( 'completed', __( 'Payment collected via cash', 'wepos' ) );
         $order->add_order_note( sprintf( __( 'Cash tendered amount: %1$s, Change amount: %2$s', 'wepos' ), $order->get_meta( '_wepos_cash_tendered_amount', true ), $order->get_meta( '_wepos_cash_change_amount', true ) ) );
```

Comment:
If possible, get some DOA by breaking into new lines and re-formatting this line.